web/assets/auto-resize-text-area.js

```diff
@@ -0,0 +1,13 @@
+const autoResizeTextArea = () => {
```

Comment:
When you get an error on a form but have entered a lot of text the textarea doesn't grow to fit until you edit it

Reply:
Fixed!